### DISCUSSION TITLE
A page/document of used open source project at GitHub?

### DISCUSSION THREAD

**Discussion by egorsmkv** [community/community#22782] [New to GitHub]
<p>Where can I see which projects from open source GitHub uses?</p>
<p>Like a document from Dropbox Paper: <a href="https://paper.dropbox.com/static/oss.pdf" rel="nofollow noopener">https://paper.dropbox.com/static/oss.pdf</a></p>

**Comment by ernest-phillips** [comment, marked as answer]
<p>Hi <a class="mention" href="/egorsmkv">@egorsmkv</a> :wave:</p>
<p>GitHub does not have an equivalent document listing the technologies we use to build our products.</p>
